docs/reference/stdlib.md

```diff
@@ -197,6 +197,11 @@ The following functions are available in Nextflow scripts:
 `branchCriteria( closure )`
 : Create a branch criteria to use with the {ref}`operator-branch` operator.
 
+`env( name )`
```

Comment:
Would not make sense to allow this also in the config?

Reply:
Likely it can be added a hook in the `ConfigBase` along the same lines of `includeConfig`

Reply:
https://github.com/nextflow-io/nextflow/blob/make_RepositoryProvider_revision_public/modules/nextflow/src/main/groovy/nextflow/config/ConfigBase.groovy#L80-L80